riff-raff.yaml

```diff
@@ -32,7 +32,7 @@ deployments:
     type: aws-lambda
     parameters:
       prefixStack: false
-      bucket: workflow-dist
+      bucketSsmLookup: true
```

Comment:
Since https://github.com/guardian/riff-raff/pull/885, `bucketSsmLookup: true` is the default, so you could omit this entirely.

Reply:
ahh yes, thanks - updated in https://github.com/guardian/pinboard/pull/195/commits/226235dc71a877e6dcc06b5054bc49ed5ca9bc89